tests/console/ovn.pm

```diff
@@ -37,7 +37,7 @@ sub run {
     assert_script_run "ovn-sbctl set-connection ptcp:6642";
 
     my $ovn_encap_ip = "";
-    if (is_tumbleweed) {
+    if (is_tumbleweed || is_sle('>=16') || is_leap('>=16.0')) {
```

Comment:
unless (is_sle('<16') || is_leap('<16.0'))

Reply:
This part use 'if  else', so I don't want to change to use 'unless'